app/modules/asset_groups/models.py

```diff
@@ -786,226 +816,298 @@ def build_detection_request(self, job_uuid, model):
             _external=True,
         )
 
-        ia_config_reader = IaConfig(current_app.config.get('CONFIG_MODEL'))
-        detector_config = ia_config_reader.get_named_detector_config(model)
-
-        assert 'start_detect' in detector_config
-
         model_config = {
-            'endpoint': detector_config['start_detect'],
```

Comment:
This is not needed.  It is sent to Sage and is ignored.  The endpoint is encoded within the URL, not the body of the request